docs/adr/0024-Scheduling.md

```diff
@@ -0,0 +1,89 @@
+# 24. Scoping dependencies inline with lifetime scope
+
+Date: 2025-01-20
+
+## Status
+
+Proposed
+
+## Context
+
+Adding the ability to schedule message (by providing `TimeSpan` or `DateTimeOffset`) give to user flexibility to `Send`, `Publis` and `Post`.
+
+
+
+## Decision
+
+Giving support to schedule message, it's necessary breaking on `IAmACommandProcessor` by adding these methods:
+
+```c#
+public interface IAmACommandProcessor
+{
+    string SchedulerSend<TRequest>(TimeSpan delay, TRequest request) where TRequest : class, IRequest;
```

Comment:
I wonder if we would be better off overloading Send with a time, than adding a new name here

Reply:
Alternatively, how about SendAt, PublishAt, PostAt etc.